packages/sdk-client-v3/src/utils/executor.ts

```diff
@@ -106,7 +105,11 @@ export default async function executor(request: HttpClientConfig) {
               return { _response, shouldRetry: true }
             }
           } catch (e) {
-            if (e.name.includes('AbortError') && retryWhenAborted) {
+            if (
+              (e.name.includes('AbortError') ||
```

Comment:
Is it possible to add a comment here to explain why we needed this two different error names and which error name belongs to which nodejs version ?

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/877/commits/fa89ac7e5e2d1b8f3cc4d1a1d16d029607439340